podcasts/New Search/Views/Results/PodcastCarouselView.swift

```diff
@@ -126,6 +126,15 @@ struct PodcastsCarouselView: View {
         }
     }
 
+    private func fillerPodcast() -> Podcast {
```

Comment:
Thoughts about renaming to placeholder? I can also see this being useful elsewhere, should we move this to a static var? Then we can just use `Podcast.placeholder` or `\.placeholder`?

Reply:
I'd rather not use this outside. 😅

I think a better solution would be to just make `Podcast` optional in the view or create a "`FillerView`".

Anyway, I think we can extract when the opportunity comes. Wdyt?

Reply:
Sounds good.